sdks/python/apache_beam/dataframe/schemas.py

```diff
@@ -275,7 +275,7 @@ def _unbatch_transform(proxy, include_indexes):
 
     return beam.ParDo(
         _UnbatchWithIndex(ctor) if include_indexes else _UnbatchNoIndex(ctor)
-    ).with_output_types(ctor)
```

Comment:
Note this is a workaround I had added, because the override for `infer_output_type` did not work (this typehint is verified in schemas_test.py).